config/user_config.yaml

```diff
@@ -0,0 +1,42 @@
+InternalTransferReferences:
```

Comment:
The config folder is intended to have only custom user configuration, not something part of the repository or source code.

If we need test configs we can add them to the test folder.

Reply:
...unintended commit